lib/aiken/math/bitwise.ak

```diff
@@ -0,0 +1,342 @@
+use aiken/builtin
+
+pub opaque type Intermediary<t> {
+  inner: Int,
+}
+
+pub const zero_intermediary = Intermediary { inner: 0 }
+
+pub const one_intermediary = Intermediary { inner: 1 }
+
+pub opaque type Scalar {
+  inner: Int,
+}
```

Comment:
How do you construct a Scalar?

Reply:
Do you like the PR overall. I'll finish the PR just wanted some feedback.